cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/ExpressionHandler.kt

```diff
@@ -352,7 +350,7 @@ class ExpressionHandler(frontend: PythonLanguageFrontend) :
             } else {
                 // try to resolve -> [ConstructExpression]
                 val currentScope = frontend.scopeManager.currentScope
-                val record =
+                /*val record =
```

Comment:
I would delete commented out code. At least so far our unwritten policy was to delete it even if we were not certain to not need it in the future